packages/config/config/devices/0x0346/alarm_keypad_gen2.json

```diff
@@ -19,302 +24,385 @@
 	"paramInformation": {
 		"1": {
 			"label": "Heartbeat Interval",
-			"description": "This parameter is the interval between heartbeats in minutes",
+			"description": "Number of minutes between heartbeats",
```

Comment:
I wasn't clear enough it seems. This description does not add any value, so it should be removed. The label already states that this is an interval between heartbeats. And the unit below states that the parameter is in minutes.

The same is true for several of the other descriptions.

Reply:
No worries, I will change that. Some of them though are probably needed but let me know if you think they are not.